ServerCore/Pages/Submissions/AuthorIndex.cshtml.cs

```diff
@@ -1,21 +1,21 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.EntityFrameworkCore;
 using ServerCore.DataModel;
 using ServerCore.ModelBases;
 
 namespace ServerCore.Pages.Submissions
 {
+    [Authorize(Policy = "IsEventAdminOrAuthorOfPuzzle")]
     public class AuthorIndexModel : EventSpecificPageModel
     {
-        private readonly ServerCore.DataModel.PuzzleServerContext _context;
+        private readonly PuzzleServerContext dbContext;
```

Comment:
Please do not rename this yet - I am putting in some standardization here, and it will be much easier to do a bulk rename once I'm done with that.